dotCMS/src/integration-test/java/com/liferay/portal/language/LanguageUtilTest.java

```diff
@@ -0,0 +1,99 @@
+package com.liferay.portal.language;
+
+import com.dotcms.datagen.LanguageDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.SystemException;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class LanguageUtilTest {
+
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4324168)